### ISSUE TITLE
pushwoosh depends on alpha builds

### ISSUE THREAD

**Issue by lwky** [Pushwoosh/pushwoosh-android-sdk#167]
pushwoosh has multiple dependencies on alpha and beta versions of libraries, instead of the stable ones

```
com.pushwoosh:pushwoosh:6.2.7
|    +--- androidx.legacy:legacy-support-v4:[1.0.0,) -> 1.0.0
|    |    +--- androidx.core:core:1.0.0 -> 1.5.0-beta01 (*)
|    |    +--- androidx.media:media:1.0.0
|    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.2.0-beta01
|    |    |    +--- androidx.core:core:1.0.0 -> 1.5.0-beta01 (*)
|    |    |    \--- androidx.versionedparcelable:versionedparcelable:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.legacy:legacy-support-core-utils:1.0.0 (*)
|    |    +--- androidx.legacy:legacy-support-core-ui:1.0.0 (*)
|    |    \--- androidx.fragment:fragment:1.0.0 (*)
|    \--- androidx.work:work-runtime:[2.0.0,) -> 2.7.0-alpha05
```

**Comment by wfhm** [comment]
@lwky,

Thanks for pointing it out! I'll check if we can use strict versioning and still allow using alpha versions and let you know about the results shortly.

**Comment by zarko-tg** [comment]
FYI, Cordova Android project(s) depending on your plugin all of a sudden doesn't build anymore due to:

```
...
> Task :app:checkReleaseAarMetadata FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkReleaseAarMetadata'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckAarMetadataWorkAction
   > The minCompileSdk (31) specified in a
     dependency's AAR metadata (META-INF/com/android/build/gradle/aar-metadata.properties)
     is greater than this module's compileSdkVersion (android-30).
     Dependency: androidx.work:work-runtime:2.7.0-beta01.
     AAR metadata file: /Users/user/.gradle/caches/transforms-3/aa434d02980a2c0e3b7f96b5367819ef/transformed/work-runtime-2.7.0-beta01/META-INF/com/android/build/gradle/aar-metadata.properties.
```

Cordova has no support for Android API 31 yet (it'll probably take some time).

We'll have to fiddle with our build pipeline again to lock androidx.work to 2.6.0, along what @STFBEE said up there.

**Comment by wfhm** [comment]
@lwky, @STFBEE,

Will be fixed with the next release.

@zarko-tg, Cordova plugin will be updated shortly as well.

**Comment by wfhm** [comment]
Fixed with 6.3.2 release.
